### ISSUE TITLE
Menu doesn't work

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#343]
Originally reported on Google Code with ID 344

```
Describe your system:
 - Operating System (be specific): Mac OSX 10.5.8
 - Machine type (32-bit or 64-bit): 32 bit
 - Mupen64Plus version: 1.99.3
 - Plugins used: Glide

For visual artifacts, give:
- game name and CRC/MD5 hash (printed on console)
- description of the artifact (missing polygons, flashing, wrong colors,
etc)

Describe the problem:
When I try to select items in the menu, I am unable to.  Using the control stick only
lets me cycle 
through the different menu screens (items, world map, etc)

Please provide any additional information below.

```

Reported by `mrkugel@yahoo.com` on 2010-03-03 01:38:08

**Comment by richard42** [comment]
```
Which menu?
If you mean a specific game then please specify which you mean. And glide64 wasnt
released of 1.99.3 yet. So please also add the revision of Glide. What Input are you
using? Have you checked what buttons are assigned?
```

Reported by `sven@narfation.org` on 2010-03-03 09:13:57

**Comment by richard42** [comment]
Reported by `richard42g` on 2010-12-01 01:30:39
- **Status changed**: `Invalid`
